diff --git a/patches/sqli_low_original.php b/patches/sqli_low_patched.php
index 6448ea4..780aa84 100644
--- a/patches/sqli_low_original.php
+++ b/patches/sqli_low_patched.php
@@ -1,24 +1,29 @@
 <?php
 
 if( isset( $_REQUEST[ 'Submit' ] ) ) {
-	// Get input
-	$id = $_REQUEST[ 'id' ];
 
-	// Check database
-	$query  = "SELECT first_name, last_name FROM users WHERE user_id = '$id';";
-	$result = mysqli_query($GLOBALS["___mysqli_ston"],  $query ) or die( '<pre>' . ((is_object($GLOBALS["___mysqli_ston"])) ? mysqli_error($GLOBALS["___mysqli_ston"]) : (($___mysqli_res = mysqli_connect_error()) ? $___mysqli_res : false)) . '</pre>' );
+    $id = $_REQUEST['id'];
 
-	// Get results
-	while( $row = mysqli_fetch_assoc( $result ) ) {
-		// Get values
-		$first = $row["first_name"];
-		$last  = $row["last_name"];
+    $stmt = mysqli_prepare(
+        $GLOBALS["___mysqli_ston"],
+        "SELECT first_name, last_name FROM users WHERE user_id = ?"
+    );
 
-		// Feedback for end user
-		$html .= "<pre>ID: {$id}<br />First name: {$first}<br />Surname: {$last}</pre>";
-	}
+    mysqli_stmt_bind_param($stmt, "i", $id);
 
-	mysqli_close($GLOBALS["___mysqli_ston"]);
+    mysqli_stmt_execute($stmt);
+
+    $result = mysqli_stmt_get_result($stmt);
+
+    while( $row = mysqli_fetch_assoc( $result ) ) {
+
+        $first = $row["first_name"];
+        $last  = $row["last_name"];
+
+        $html .= "<pre>ID: {$id}<br />First name: {$first}<br />Surname: {$last}</pre>";
+    }
+
+    mysqli_stmt_close($stmt);
 }
 
-?>
+?>
\ No newline at end of file
